### PR TITLE
Add a Custom Images section to the install file

### PR DESCRIPTION
**Pull request by nievesmontero**
https://github.com/containers/toolbox/issues/1102

Signed-off-by: Nieves Montero <nmontero@redhat.com>

### CODE REVIEW

**Comment by travier** [comment]
This does not work properly: https://containertoolbx.org/install/containerfile.md

File: install.md [hunk 1]
```diff
@@ -53,4 +53,37 @@ It can be built and installed as any other typical Meson-based project:
 
 Toolbx is written in Go. Consult the [src/go.mod](https://github.com/containers/toolbox/blob/main/src/go.mod) file for a full list of all the Go dependencies.
 
-By default, Toolbx uses Go modules and all the required Go packages are automatically downloaded as part of the build. There's no need to worry about the Go dependencies, unless the build environment doesn't have network access or any such peculiarities.
\ No newline at end of file
+By default, Toolbx uses Go modules and all the required Go packages are automatically downloaded as part of the build. There's no need to worry about the Go dependencies, unless the build environment doesn't have network access or any such peculiarities.
+
+## Custom Images
+There are two different ways of creating a custom image. The first one consists on starting from scratch by creating a container file or using an existing one. In the following link, we can see an example of a [Containerfile](containerfile.md). The other possibility consists on customizing an existing image.
```
Comment (travier) [inline install.md:59]: Not the correct name here

Comment (travier) [reply install.md:59]: Fixed in https://github.com/containers/containertoolbx.org/commit/299a2693c8d40f7416367f4aafaf3f70a74bf670

Comment (debarshiray) [reply install.md:59]: Thanks for keeping an eye on things, @travier - marking as resolved because @jimmac already fixed this.

**Pull request merged**